src/plugins/embeddable/public/lib/panel/embeddable_panel.tsx

```diff
@@ -430,27 +430,28 @@ export class EmbeddablePanel extends React.Component<Props, State> {
   };
 
   private getActionContextMenuPanel = async () => {
-    let regularActions =
+    const regularActions =
       (await this.props.getActions?.(CONTEXT_MENU_TRIGGER, {
         embeddable: this.props.embeddable,
       })) ?? [];
 
     const { disabledActions } = this.props.embeddable.getInput();
+
+    let allActions = regularActions.concat(
+      Object.values(this.state.universalActions || {}) as Array<Action<object>>
+    );
     if (disabledActions) {
       const removeDisabledActions = removeById(disabledActions);
-      regularActions = regularActions.filter(removeDisabledActions);
+      allActions = allActions.filter(removeDisabledActions);
```

Comment:
Because drilldowns are considered a "universal action," I had to do the filtering of disabled actions after the universal actions were added to the action list so that I could filter the drilldown action out of the link panel.